include/psa/crypto.h

```diff
@@ -411,8 +414,10 @@ psa_status_t psa_open_key(psa_key_id_t id,
  * Closing the key handle makes the handle invalid, and the key handle
  * must not be used again by the application.
  *
- * If the key is currently in use in a multipart operation, then closing the
- * last remaining handle to the key will abort the multipart operation.
+ * \note If the key handle was used to setup an active
```

Comment:
Nit: I don't think “setup” is accepted as a verb yet: it should be “set up”.

In fact https://english.stackexchange.com/questions/93243/login-and-payoff-are-nouns-but-can-they-be-used-as-verbs makes some good points: if you replace “an active multipart operation” by “it”, the sentence becomes “… to set it up”, not “\*to setup it”, so there's still a space when the complement isn't a pronoun.

Reply:
Thanks. That briefly crossed my mind while writing it, but didn't return to re-read it.